docs: explain static vs dynamic meta tags more clearly (#41215)
expo/expo: docs/pages/router/web/static-rendering.mdx

}
```

> For this to work correctly, make sure to wrap your root _layout.tsx with `<Head.Provider>`:

Reviewer: ```suggestion
Then, wrap the root layout navigator (**app/_layout.tsx) with `<Head.Provider>`:
```

This context should be concise and meaningful. The developer won't the context about correct/incorrect usage when the come across this for the first time.

```tsx app/_layout.tsx
import { Stack } from 'expo-router';
import Head from 'expo-router/head';

> export default function StackLayout() {

Reviewer: ```suggestion
export default function RootLayout() {
```
